src/Activation/AcceptTermsOfService.tsx

```diff
@@ -26,13 +26,14 @@ import {
 
 const AcceptTermsOfService: FunctionComponent = () => {
   useStatusBarEffect("dark-content", Colors.background.primaryLight)
+  const { t } = useTranslation()
   const configuration = useConfigurationContext()
+  const { goToNextScreen } = useActivationNavigation()
+
   const [boxChecked, toggleCheckbox] = useState(false)
-  const { t } = useTranslation()
-  const navigation = useNavigation()
 
   const handleOnPressNext = () => {
-    navigation.navigate(ActivationStackScreens.ActivateExposureNotifications)
+    goToNextScreen("AcceptTermsOfService")
```

Comment:
what do you think about:
```suggestion
    goToNextScreenFrom("AcceptTermsOfService")
```
?

Reply:
Good call!